### DISCUSSION TITLE
Exclude repos from organization level webhook

### DISCUSSION THREAD

**Discussion by mkurz** [community/community#36180] [Apps, API and Webhooks]
We have quite a lot of repositories within our organization. We want to set up an identical webhook for almost all repos, except for a few...
Right now, on organization level I can not exclude / include / whitelist / blacklist repositories...
Would be nice if you give users the choice to have various options like:
* _All_ repos _except_ a selected list of repos (that means also newly created or transfered repos will have the webhook enabled unless I add them to that blacklist manually)
* Just selected repos (so that is a fixed list, meaning newly created / transferred repos will not be selected by default).

**Comment by justdave** [comment]
Since I needed this feature, I wrote a webhook proxy that'll filter by repo and forward the requests to Discord. https://github.com/bugzilla/github-discord-relay

**Comment by Beanow** [comment]
I've added a similar suggestion, to filter by privacy setting of the repositories: #67380

**Comment by jakirkham** [comment]
Also have this use case. Currently filter on the side of service that receives these webhooks, but it makes the GitHub activity way noisier than it needs to be and it causes unnecessary churn for the service. Would be nice to exclude a handful of repos as needed
